frontend/app/services/subscription-service.server.ts

```diff
@@ -23,15 +23,17 @@ type SubscriptionInfo = z.infer<typeof subscriptionInfoSchema>;
 export const getSubscriptionService = moize(createSubscriptionService, { onCacheAdd: () => log.info('Creating new subscription service') });
 
 function createSubscriptionService() {
+  const { CDCP_API_BASE_URI } = getEnv();
+
   async function getSubscription(sin: string) {
     const auditService = getAuditService();
     const instrumentationService = getInstrumentationService();
     auditService.audit('alert-subscription.get', { sin });
-
+    //76c48130-e1d4-4c2f-8dd0-1c17f9bbb4f6
     // TODO: "IT-Security won't like SIN being passed as identifier"
-    // TODO: add CDCP_API_BASE_URI
+
     const userId = sin;
-    const url = new URL(`https://api.example.com/v1/users/${userId}/subscriptions`);
+    const url = new URL(`${CDCP_API_BASE_URI}/api/v1/users/${userId}/subscriptions`);
```

Comment:
We should be using HATEOAS' `links` to retrieve subscriptions for a user rather than hard-coding the subscriptions endpoint. This can be done by calling `/api/v1/users/{userId}`, then using the `subscriptions` link for a subsequent API call.

Reply:
^ That way, this service function can construct and return all the data elements that the front end would need including the user's email.